modules/mvt/src/lib/parse-mvt.js

```diff
@@ -88,6 +105,19 @@ function getDecodedFeature(feature, options = {}) {
   return decodedFeature;
 }
 
+function getDecodedFeatureBinary(feature, options = {}) {
+  const decodedFeature = feature.toLines(
```

Comment:
Maybe something like `feature.toBinaryCoordinates` would be a clearer name?

Reply:
Done